sdk/orm-django/django_encryption/fields.py

```diff
@@ -53,33 +210,35 @@ def __init__(
             expiration_secs: Optional[int] = None,
             data_type_name: Optional[str] = None,
             on_error: Any = None,
+            eager: bool = True,
             **kwargs):
         self._vault_property: Optional[str] = vault_property
```

Comment:
nit: why is only _vault_property marked as optional?

Reply:
Fixed (it was there because of historical reason. It's not incorrect, just unnecessary)